_includes/developer_tags.html

```diff
@@ -0,0 +1,16 @@
+<div class="dropdown-menu" id="dropdownFilterTags">
```

Comment:
Are you able to combine this file with `research_tags` into just one `includes` file and pass parameters to each to differentiate